CMakeLists.txt

```diff
@@ -26,12 +36,12 @@ if(APPLE)
 endif(APPLE)
 
 # includes and source files
-include_directories(include)
-include_directories(src)
+file(GLOB_RECURSE SWMM_HEADERS include/*.h src/*.h)
 file(GLOB SWMM_SOURCES src/*.c)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 
 # library
-add_library(swmm5 SHARED ${SWMM_SOURCES})
+add_library(swmm5 SHARED ${SWMM_SOURCES} ${SWMM_HEADERS})
```

Comment:
Why was this needed?

Reply:
It’s a nice feature in visual studio. When you generate the solution; it separates the project related header files into a clean directly inside the project. Alternately, the headers all get dumped into the external dependencies directly.

Reply:
probably good for other IDEs as well; I know in Xcode the generated project only has references to the source files you add here. So if the headers aren't here, then they aren't easily accessible.

Reply:
Ok, sounds good